public/js/forecast.js

```diff
@@ -0,0 +1,13 @@
+const forecastGetter = async (location) => {
+	const response = location.coords ? await fetch(`/weather?lat=${location.coords.latitude}&lon=${location.coords.longitude}`) : await fetch(`/weather?address=${location}`)
+	const data = await response.json()
+	forecastMarkupCreator(data)
+}
+
+const forecastMarkupCreator = (data) => {
+	if (data.error) return weatherMessage.textContent = data.error
+	weatherIcon.style.display = 'block'
```

Comment:
empty line between return and next line.
tabulation.

Reply:
Fixed.

Reply:
Fixed.